.circleci/config.yml

```diff
@@ -106,6 +106,20 @@ jobs:
           command: |
             cd telemetry-airflow
             bash bin/test-parse
+  verify-dags-up-to-date:
+    docker: *docker
+    steps:
+    - checkout
+    - *build
+    - run: 
+        name: Generate DAGs
+        command: |
+          mv dags/ existing_dags/
+          mkdir dags
+          PATH="venv/bin:$PATH" script/generate_airflow_dags
+    - run:
+        name: Verify that DAGs were correctly generated and are up-to-date
+        command: diff -aq existing_dags/ dags/
```

Comment:
`diff` returns an exit code 1 if there are differences between files

Reply:
Under `verify-requirements`, we generate `requirements.txt` in place and use `git diff --exit-code` to verify that nothing has changed. Unless there's a clear benefit to using `diff -aq` here, I'd like to see us use a consistent methodology for verifying generated content.